.circleci/config.yml

```diff
@@ -172,19 +172,22 @@ step-install-nodejs-on-mac: &step-install-nodejs-on-mac
   run:
     name: Install Node.js 10 on MacOS
     command: |
-      if [ "`uname`" == "Darwin" ]; then
-        brew update
-        brew install node@10
-        echo 'export PATH="/usr/local/opt/node@10/bin:$PATH"' >> $BASH_ENV
-      fi
+        if [ "`uname`" == "Darwin" ]; then
+          curl -sSLf https://get.notionjs.com | bash
+          export NOTION_HOME="$HOME/.notion"
+          source "$NOTION_HOME/load.sh"
+          notion install node 10
+          echo 'export NOTION_HOME="$HOME/.notion"' >> $BASH_ENV
+          echo 'source "$NOTION_HOME/load.sh"' >> $BASH_ENV
+          echo 'export PATH="$PATH:'$NOTION_HOME'/bin"' >> $BASH_ENV
+        fi
 
 step-install-gnutar-on-mac: &step-install-gnutar-on-mac
   run:
     name: Install gnu-tar on macos
     command: |
       if [ "`uname`" == "Darwin" ]; then
-        brew update
-        brew install gnu-tar
+        HOMEBREW_NO_AUTO_UPDATE=1 brew install gnu-tar
```

Comment:
This also makes `brew install` faster (we don't need to update to install this, it's already in the brew local)